int32[] encoder_count
int32[] input_voltage
int32[] motor_velocity
int32[] motor_current
